.github/workflows/build.yml

```diff
@@ -276,7 +283,7 @@ jobs:
           git config --global --add safe.directory /github/workspace/sources/nuttx
           git config --global --add safe.directory /github/workspace/sources/apps
           cd sources/nuttx/tools/ci
-          ./cibuild.sh -g -i -A -C -R testlist/${{matrix.boards}}.dat
+          ./cibuild.sh -g -i -A -C -N -R -S testlist/${{matrix.boards}}.dat
```

Comment:
Please add description of those flags in the PR summary.

Reply:
I believe it comes from here? https://github.com/apache/nuttx/pull/13301

Reply:
Yes @lupyuen  I will comment on the Pr with the requested info as soon as the CI is ok

Reply:
@cederom @lupyuen done